### DISCUSSION TITLE
Disallowed variable names in template - configDescription

### DISCUSSION THREAD

**Discussion by JamesMHenderson** [renovatebot/renovate#30969] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Running version 38.51.0 I see the following warning in the logs

INFO: Disallowed variable names in template
       "varNames": ["configDescription"],
       "template": "{{{configDescription}}}"
       
configDescription is part of the default template for dependencyDashboardFooter and prBodyTemplate

### Logs (if relevant)

<details><summary>Logs</summary>

```
INFO: Disallowed variable names in template
       "varNames": ["configDescription"],
       "template": "{{{configDescription}}}"

```

**Comment by RahulGautamSingh** [comment, marked as answer]
For `dependencyDashboardFooter` [these](https://docs.renovatebot.com/templates/) temaplte fields are available and `configDescription` is not one of them, which is causing the error in your case.

`configDescription` is available for `prBodyTemplate` though so you can continue using it there.

**Reply by JamesMHenderson** [reply]
Thanks, I've taken another look at my config, turns out I had a env variable setting the `dependencyDashboardFooter` which I missed
